core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -3298,7 +3300,7 @@ object ZIO extends ZIOCompanionPlatformSpecific {
 
   final class TimeoutTo[R, E, A, B](self: ZIO[R, E, A], b: B) {
     def apply[B1 >: B](f: A => B1)(duration: Duration): ZIO[R with Clock, E, B1] =
-      (self map f) raceFirst (ZIO.sleep(duration).interruptible as b)
+      (self map f) raceFirst (ZIO.sleep(duration) as b)
```

Comment:
I think there's no reason not to make it interruptible, even with disconnect - disconnect will still leak the fiber.

Reply:
Yes, you're right. Will change.